doc/classes/TreeItem.xml

```diff
@@ -63,6 +63,16 @@
 				Removes all OpenType features.
 			</description>
 		</method>
+		<method name="create_child">
+			<return type="TreeItem">
+			</return>
+			<argument index="0" name="idx" type="int" default="-1">
+			</argument>
+			<description>
+				Creates an item adds it as a child.
+				The new item will be the [code]idx[/code]th child, or it will be the last child if there are not enough children.
```

Comment:
```suggestion
				The new item will be inserted as position [code]idx[/code] (the default value [code]-1[/code] means the last position), or it will be the last child if [code]idx[/code] is higher than the child count.
```